tests/ecs.rs

```diff
@@ -0,0 +1,27 @@
+//#![cfg(feature = "ecs")]
```

Comment:
fixed. I had commented this out will trying to figure out the cmdline flags to run the tests. I added it back